### ISSUE TITLE
Text that appears “blurred out” visually is fully readable by a screen reader.

### ISSUE THREAD

**Issue by msftedad** [microsoft/vsts-extension-retrospectives#1195]
#A11y #accessibility

**Describe the bug**
Text that appears “blurred out” visually is fully readable by a screen reader. Users relying on screen readers that read blurred-out text can lead to unintended access to sensitive or confidential information. 

**Are you using the Retrospectives Tool on-prem?**
No

**To Reproduce**

1. open the URL https://dev.azure.com/eDADPlayground/SCRUMProject .
2. Expand left navigation and select boards-->Retrospectives.
3. Now under the 'board' tab we have different feedback groups.
4. verify whether Text that appears “blurred out” visually is fully readable by a screen reader or not.

**Expected behavior**
Blurred text should be concealed or rendered inaccessible to the screen reader to maintain privacy and confidentiality.

**Screenshots**

https://github.com/user-attachments/assets/c1f838ae-1d1f-4786-a3b7-e23ac6d51bdc

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/d028a898-b69c-45ea-ab42-253590af3f18" />

**What screen type are you seeing this on ?**
[ ] Desktop Windows 11

**Additional context**

**Comment by polatengin** [comment]
fixed with PRs
